src/test/java/org/gridsuite/study/server/LoadFlowTest.java

```diff
@@ -643,6 +644,9 @@ private void createOrUpdateParametersAndDoChecks(UUID studyNameUserIdUuid, Strin
 
         message = output.receive(TIMEOUT, elementUpdateDestination);
         assertEquals(studyNameUserIdUuid, message.getHeaders().get(NotificationService.HEADER_ELEMENT_UUID));
+        message = output.receive(TIMEOUT, studyUpdateDestination);
+        assertEquals(UPDATE_TYPE_COMPUTATION_PARAMETERS, message.getHeaders().get(NotificationService.HEADER_UPDATE_TYPE));
+
```

Comment:
could assert that it's certainly LOADFLOW in HEADER_COMPUTATION_TYPE here as well